lean4-infoview/src/infoview/messages.tsx

```diff
@@ -188,13 +188,14 @@ export function useMessagesForFile(uri: DocumentUri, line?: number): Interactive
                 if (err?.code === -32801) {
                     // Document has been changed since we made the request.
                     // This can happen while typing quickly, so server will catch up on next edit.
+                    console.log('getInteractiveDiagnostics: document has been changed since we made the request ')
```

Comment:
Similarly this log, we often take this branch so it's a lot of debug output.

Reply:
again this is only vscode extension debug console output, user's will not see this stuff.

Reply:
Yep I know, sorry, 'production' was maybe not the best choice of word. What I mean is that I'm not sure these messages are useful even for us developers outside of debugging this particular PR, so I would remove them as leftover debug statements.

Reply:
Well it helped me find the very thing that caused the bug I fixed in messages.tsx - but it is not that noisy, I don't see this message very often, usually only during startup of a new editing session.  This is what a typical debug session console output looks like these days:
![image](https://user-images.githubusercontent.com/18707114/175384809-b1e8c60e-23fd-4bec-98be-de5dfe77e2ab.png)

then nothing more while editing the .lean file unless I do a command like Restart Server, or change the version.  The biggest  noise in the console is from things like this that pop up randomly:

![image](https://user-images.githubusercontent.com/18707114/175385093-15e01125-5797-4993-924a-ea27f80f9e33.png)

Reply:
But I'll agree to remove this one, since I can't guarantee it doesn't get noisy in some edge case I haven't found.

Reply:
Hmmm, the 'Got unexpected error form' looks like another bug.